### PR TITLE
Clarify that subject references a separate DAG

### PR DESCRIPTION
**Pull request by sudo-bmitch**
This is resolving concerns raised in https://github.com/opencontainers/distribution-spec/pull/502 and discussed on today's meeting.

### CODE REVIEW

File: image-index.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 - **`subject`** *[descriptor](descriptor.md)*
 
     This OPTIONAL property specifies a [descriptor](descriptor.md) of another manifest.
-    This value, used by the [`referrers` API][referrers-api], indicates a relationship to the specified manifest.
+    This value defines a loose relationship to a separate DAG structure, and is used by the [`referrers` API][referrers-api] to this manifest in the list of responses for the subject digest.
```
Comment (rchincha) [inline image-index.md:96]: "a weak association"?

^ to be more formal

Comment (sajayantony) [reply image-index.md:96]: If you are referencing a DAG, then maybe you can qualify it similar to 
https://github.com/opencontainers/image-spec/blob/970322241ad9c8868d15e6859c9b446ad06b944d/descriptor.md?plain=1#L3

Comment (sudo-bmitch) [reply image-index.md]: Both suggestions applied. Thanks!

**Review by sajayantony — APPROVED** [review]
/cc @mikebrow

**Review by tianon — APPROVED** [review]

**Comment by tianon** [comment]
@mikebrow if this isn't enough to resolve your concerns, can you submit a follow up? 🙇❤️

**Comment by mikebrow** [comment]
@tianon will do.. LGTM

**Pull request merged**